Fix comments by JH (part 2) (#355)
cfrg/draft-irtf-cfrg-voprf: draft-irtf-cfrg-voprf.md

- I2OSP(x, xLen): Converts a non-negative integer `x` into a byte array
  of specified length `xLen` as described in {{!RFC8017}}. Note that
  this function returns a byte array in big-endian byte order.
- The notation `T U[N]` refers to an array called U containing N items of type
  T. The type `opaque` means one single byte of uninterpreted data. Items of
  the array are zero-indexed and referred as `U[j]` such that 0 <= j < N.

All algorithms and procedures described in this document are laid out
in a Python-like pseudocode. Each function takes a set of inputs and parameters
and produces a set of output values. Parameters become constant values once the
protocol variant and the ciphersuite are fixed.

> The `PrivateInput` data type refers to information that must be kept in
> secrecy, and the `PublicInput` data type indicates that it is not required to
> keep the information secret. These data types are opaque byte strings of
> arbitrary length no larger than 2^13 octets.

Reviewer: ```suggestion
The `PrivateInput` data type refers to inputs that are known only to the client
in the protocol, whereas the `PublicInput` data type refers to inputs that are 
known to both client and server in the protocol. Both `PrivateInput` and
`PublicInput` are opaque byte strings of arbitrary length no larger than 2^13 octets.
```

            "Challenge"

  expectedC = G.HashToScalar(h2Input)

>   verified = expectedC == c

Reviewer: ```suggestion
  verified = (expectedC == c)
```

~~~
Input:

  Scalar skS
  Element blindedElement

Output:

  Element evaluatedElement

def BlindEvaluate(skS, blindedElement):
  evaluatedElement = skS * blindedElement
  return evaluatedElement
~~~

Servers send the output `evaluatedElement` to clients for processing.
Recall that servers may process multiple client inputs by applying the
`BlindEvaluate` function to each `blindedElement` received, and returning an

> array with the corresponding `evaluatedElement`s.

Reviewer: ```suggestion
array with the corresponding `evaluatedElement` values.
```

information about the client's private input x from an invocation of the
protocol.

> Essentially, input secrecy tells us that, even if the server learns

Reviewer: ```suggestion
Essentially, input secrecy is the property that, even if the server learns
```

> the client's private input x at some point in the future, then the server will
> not be able to link any particular PRF evaluation to x. This property is
> also known as unlinkability {{DGSTV18}}.

Reviewer: ```suggestion
the client's private input x at some point in the future, the server cannot
link any particular PRF evaluation to x. This property is also known as unlinkability {{DGSTV18}}.
```

## Domain Separation {#domain-separation}

> Applications MUST construct input to the protocol to provide domain

Reviewer: ```suggestion
Applications SHOULD construct input to the protocol to provide domain
```

Reviewer: A MUST isn't enforceable.
